x-pack/filebeat/module/googlecloud/audit/config/pipeline.js

```diff
@@ -38,6 +38,7 @@ function Audit(keep_original_message) {
     var saveMetadata = new processor.Convert({
         fields: [
             {from: "json.logName", to: "log.logger"},
+            {from: "json.insertId", to: "event.id"},
```

Comment:
The pubsub input [sets this field](https://github.com/elastic/beats/blob/master/x-pack/filebeat/input/googlepubsub/input.go#L198) already but this is a better value.